docs/topics/db/aggregation.txt

```diff
@@ -323,6 +323,17 @@ title that starts with "Django" using the query::
 
     >>> Book.objects.filter(name__startswith="Django").aggregate(Avg('price'))
 
+When filtering against :ref:`multiple values <spanning-multi-valued-relationships>`,
+be careful to structure your filters according to the behavior you desire::
+
+    # Annotated list of all authors with books with between 100 and 200 pages.
+    >>> Author.objects.filter(books__pages__lt=200,
```

Comment:
I think it would be more readable with this style:
```
Author.objects.filter(
    books__pages__lt=200,
    books__pages__gt=100,
).annotate(...)
```
You can do something similar with the second example too. Please squash commits when updating and I'll make any final tweaks needed. Thanks!